bb_wrapper/models/pagamentos.py

```diff
@@ -86,16 +87,24 @@ def _set_data(cls, values):
             values["email"] = key
         elif key_type == TipoChavePIX.uuid:
             values["identificacaoAleatoria"] = key
-        elif key_type == TipoChavePIX.documento:
-            key_value = cpfcnpj.clear_punctuation(key)
-            if len(key_value) == 11:
-                values["cpf"] = key_value
-            else:
-                values["cnpj"] = key_value
```

Comment:
pode manter essa verificação e adição da chave através de uma chave CPF/CNPJ no valor `chave` também.

Consequentemente quando for enviado uma chave CPF/CNPJ no campo `chave` e for enviado também um CPF/CNPJ no campo `documento`, o valor do campo chave será sobrescrito.

Mas acredito que isso é o ideal para não alterar a interface dos argumentos do método no wrapper.